features/partners.feature

```diff
@@ -286,61 +288,7 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
-      | alles leeg en onbekende geslachtsnaam   |          | .             |               |                |              |                   |               |                |              |           |                 | PartnerOnbekend | true     |
-      | alleen standaardwaarden                 |          | .             | 00000000      | 0000           | 0000         | .                 | 00000000      | 0000           | 0000         |           |                 | PartnerOnbekend | true     |
-      | alleen standaardwaarde geboortedatum    |          |               | 00000000      |                |              |                   |               |                |              |           |                 | PartnerOnbekend | true     |
-      | alleen standaardwaarde verbintenis      |          |               |               |                |              | .                 |               |                |              |           |                 | PartnerOnbekend | true     |
-      | alles leeg en partner in onderzoek      |          | .             |               |                |              |                   |               |                |              | 050000    | 20220311        | PartnerOnbekend | true     |
-      | alles leeg en naam partner in onderzoek |          | .             |               |                |              |                   |               |                |              | 050200    | 20220311        | PartnerOnbekend | true     |
-      | geslacht onbekend                       | O        | .             |               |                |              |                   |               |                |              |           |                 | Partner         |          |
-      | alleen soort verbintenis                |          |               |               |                |              | H                 |               |                |              |           |                 | Partner         |          |
-      | alleen jaar datum aangaan               |          |               |               |                |              |                   | 19730000      |                |              |           |                 | Partner         |          |
-      | alleen jaar geboortedatum               |          |               | 19640000      |                |              |                   |               |                |              |           |                 | Partner         |          |
 
-      # ook PartnerOnbekend bij waarde van gealachtsaanduiding, aangaanHuwelijkPartnerschap of soortVerbintenis? Zie https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/940#issuecomment-1069293224
 
     @proxy
```

Comment:
hadden we dan niet ook een scenario willen hebben dat inderdaad alle gegevens leeg of standaardwaarde zijn? Bijvoorbeeld fields=partners en geslachtsnaam='.', overige velden zijn leeg

Reply:
Staat in mijn versie wel.